dcm2niix_gear/dcm2niix/arrange.py

```diff
@@ -138,28 +258,50 @@ def extract_archive_contents(archive_obj, work_dir):
         dcm2niix_input_dir, dirname = setup_dcm2niix_input_dir(filename, work_dir)
         archive_obj.extractall(dcm2niix_input_dir)
 
-    elif len(subdirs) == 1:
+    elif len(subdirs) >= 1:
 
         # Subdirectory name will be used as the dcm2niix input directory name
+        print(f"subdirs: {subdirs}")
```

Comment:
log.info or log.debug should be used here.

Reply:
👍